src/usermod.c

```diff
@@ -2056,26 +2055,28 @@ static void move_mailbox (void)
 		(void) snprintf (mailfile, len, "%s/%s",
 	    	             maildir, user_name);
 	}
-	mailfile[len-1] = '\0';
```

Comment:
This seems correct, but looks a bit sneaky in this patch.  Maybe mention it in the message, but I think a separate commit dropping redundant stores may be better.  There's another case below.